core/src/main/kotlin/com/malinskiy/marathon/cache/test/serialization/TestResultEntryWriter.kt

```diff
@@ -4,45 +4,42 @@ import com.malinskiy.marathon.cache.CacheEntryWriter
 import com.malinskiy.marathon.device.DeviceInfo
 import com.malinskiy.marathon.execution.Attachment
 import com.malinskiy.marathon.execution.TestResult
-import io.ktor.util.cio.*
-import io.ktor.utils.io.*
 import io.ktor.utils.io.core.*
+import io.ktor.utils.io.streams.*
+import java.io.DataOutputStream
 import java.io.File
+import java.io.OutputStream
 
 class TestResultEntryWriter(private val testResult: TestResult) : CacheEntryWriter {
 
-    override suspend fun writeTo(output: ByteWriteChannel) {
-        output.writeDeviceInfo(testResult.device)
-        output.writeInt(testResult.status.ordinal)
-        output.writeLong(testResult.startTime)
-        output.writeLong(testResult.endTime)
-        output.writeString(testResult.batchId)
-        output.writeString(testResult.stacktrace)
+    override fun writeTo(output: OutputStream) {
+        DataOutputStream(output).use {
```

Comment:
Is it expected that we allow only one `writeTo` call to the provided stream? I wouldn't close it